apps/server/src/shared/domain/interface/account.ts

```diff
@@ -0,0 +1,13 @@
+export interface IAccount {
+	id?: string;
```

Comment:
I don't if it's necessary, but we have a kind of value object type `EntityId` for entity identifiers, which wraps the raw string type.